lib/stat_tracker.rb

```diff
@@ -462,6 +462,78 @@ def biggest_team_blowout(team_id)
       diff = (game.home_goals - game.away_goals).abs
       biggest_blowout = diff if diff > biggest_blowout
     end
-    biggest_blowout
+    biggest_blowout  end
+
+  
+    def head_to_head(team_id)
+    home_team_games = get_games(team_id.to_i, :home)
+    away_team_games = get_games(team_id.to_i, :away)
+    opponents = []
+    head_to_head_hash = {}
+
+    home_team_games.each do |game|
+      if !opponents.include?(game.away_team_id)
+        opponents << game.away_team_id
+      end
+    end
+ 
+    away_team_games.each do |game|
+      if !opponents.include?(game.home_team_id)
+        opponents << game.home_team_id
+      end
+    end
+  
+    # require 'pry'; binding.pry
```

Comment:
Please remove this pry binding prior to merge

Reply:
removed